diff --git a/node_modules/kontra/kontra.js b/node_modules/kontra/kontra.js
index 9b4b813..e2b6124 100644
--- a/node_modules/kontra/kontra.js
+++ b/node_modules/kontra/kontra.js
@@ -4,7 +4,7 @@
  */
 var kontra = (function () {
 
-  let noop = () => {};
+  let noop = () => { };
 
   // style used for DOM nodes needed for screen readers
   let srOnlyStyle =
@@ -721,7 +721,7 @@ var kontra = (function () {
         imageAssets[getName(url)] =
           imageAssets[resolvedUrl] =
           imageAssets[fullUrl] =
-            this;
+          this;
         emit('assetLoaded', this, url);
         resolve(this);
       };
@@ -779,8 +779,8 @@ var kontra = (function () {
             playableSource
               ? playableSource
               : canPlay[getExtension(source)]
-              ? source
-              : null,
+                ? source
+                : null,
           0
         ); // 0 is the shortest falsy value
 
@@ -800,7 +800,7 @@ var kontra = (function () {
         audioAssets[getName(url)] =
           audioAssets[resolvedUrl] =
           audioAssets[fullUrl] =
-            this;
+          this;
         emit('assetLoaded', this, url);
         resolve(this);
       });
@@ -860,7 +860,7 @@ var kontra = (function () {
         dataAssets[getName(url)] =
           dataAssets[resolvedUrl] =
           dataAssets[fullUrl] =
-            response;
+          response;
         emit('assetLoaded', response, url);
         return response;
       });
@@ -899,8 +899,8 @@ var kontra = (function () {
         return extension.match(imageRegex)
           ? loadImage(asset)
           : extension.match(audioRegex)
-          ? loadAudio(asset)
-          : loadData(asset);
+            ? loadAudio(asset)
+            : loadData(asset);
       })
     );
   }
@@ -1745,7 +1745,7 @@ var kontra = (function () {
     }
     // @endif
 
-    _pc() {}
+    _pc() { }
   }
 
   /**
@@ -2101,7 +2101,7 @@ var kontra = (function () {
      * @memberof GameObject
      * @function draw
      */
-    draw() {}
+    draw() { }
 
     /**
      * Sync property changes from the parent to the child
@@ -2863,8 +2863,8 @@ var kontra = (function () {
         textAlign == 'right'
           ? this.width
           : textAlign == 'center'
-          ? (this.width / 2) | 0
-          : 0;
+            ? (this.width / 2) | 0
+            : 0;
       // @endif
 
       this._s.map((str, index) => {
@@ -3114,12 +3114,12 @@ var kontra = (function () {
         rect.left +
         (getPropValue(_s, 'border-left-width') +
           getPropValue(_s, 'padding-left')) *
-          transformScaleX,
+        transformScaleX,
       offsetY:
         rect.top +
         (getPropValue(_s, 'border-top-width') +
           getPropValue(_s, 'padding-top')) *
-          transformScaleY
+        transformScaleY
     };
   }
 
@@ -3759,28 +3759,28 @@ var kontra = (function () {
      * @memberof Button
      * @function onEnable
      */
-    onEnable() {}
+    onEnable() { }
 
     /**
      * Function called when then button is disabled. Override this function to have the button do something when disabled.
      * @memberof Button
      * @function onDisable
      */
-    onDisable() {}
+    onDisable() { }
 
     /**
      * Function called when then button is focused. Override this function to have the button do something when focused.
      * @memberof Button
      * @function onFocus
      */
-    onFocus() {}
+    onFocus() { }
 
     /**
      * Function called when then button is blurred. Override this function to have the button do something when blurred.
      * @memberof Button
      * @function onBlur
      */
-    onBlur() {}
+    onBlur() { }
 
     onDown() {
       if (!this.disabled) {
@@ -4163,8 +4163,8 @@ var kontra = (function () {
     let pads = navigator.getGamepads
       ? navigator.getGamepads()
       : navigator.webkitGetGamepads
-      ? navigator.webkitGetGamepads
-      : [];
+        ? navigator.webkitGetGamepads
+        : [];
 
     for (let i = 0; i < pads.length; i++) {
       let gamepad = pads[i];
@@ -4511,8 +4511,8 @@ var kontra = (function () {
             ? 'left'
             : 'right'
           : y < 0
-          ? 'up'
-          : 'down';
+            ? 'up'
+            : 'down';
       }
     },
     pinch: {
@@ -4847,8 +4847,8 @@ var kontra = (function () {
         this.flow == 'column'
           ? 1
           : this.flow == 'row'
-          ? children.length
-          : this.numCols);
+            ? children.length
+            : this.numCols);
 
       let row = 0;
       let col = 0;
@@ -5513,9 +5513,6 @@ var kontra = (function () {
 
   /**
    * A fast and memory efficient [object pool](https://gameprogrammingpatterns.com/object-pool.html) for sprite reuse. Perfect for particle systems or SHUMPs. The pool starts out with just one object, but will grow in size to accommodate as many objects as are needed.
-   *
-   * <canvas width="600" height="200" id="pool-example"></canvas>
-   * <script src="assets/js/pool.js"></script>
    * @class Pool
    *
    * @param {Object} properties - Properties of the pool.
@@ -5654,7 +5651,7 @@ var kontra = (function () {
     update(dt) {
       let obj;
       let doSort = false;
-      for (let i = this.size; i--; ) {
+      for (let i = this.size; i--;) {
         obj = this.objects[i];
 
         obj.update(dt);
@@ -5676,7 +5673,7 @@ var kontra = (function () {
      * @function render
      */
     render() {
-      for (let i = this.size; i--; ) {
+      for (let i = this.size; i--;) {
         this.objects[i].render();
       }
     }
@@ -6382,7 +6379,7 @@ var kontra = (function () {
      * @memberof Scene
      * @function onShow
      */
-    onShow() {}
+    onShow() { }
 
     /**
      * Function called when the scene is hidden. Override this function to have the scene do something when hidden, such as cleaning up input events.
@@ -6399,7 +6396,7 @@ var kontra = (function () {
      * @memberof Scene
      * @function onHide
      */
-    onHide() {}
+    onHide() { }
   }
 
   function factory$2() {
diff --git a/node_modules/kontra/kontra.mjs b/node_modules/kontra/kontra.mjs
index 9059d3f..cad40de 100644
--- a/node_modules/kontra/kontra.mjs
+++ b/node_modules/kontra/kontra.mjs
@@ -2,7 +2,7 @@
  * @preserve
  * Kontra.js v9.0.0
  */
-let noop = () => {};
+let noop = () => { };
 
 // style used for DOM nodes needed for screen readers
 let srOnlyStyle =
@@ -719,7 +719,7 @@ function loadImage(url) {
       imageAssets[getName(url)] =
         imageAssets[resolvedUrl] =
         imageAssets[fullUrl] =
-          this;
+        this;
       emit('assetLoaded', this, url);
       resolve(this);
     };
@@ -777,8 +777,8 @@ function loadAudio(url) {
           playableSource
             ? playableSource
             : canPlay[getExtension(source)]
-            ? source
-            : null,
+              ? source
+              : null,
         0
       ); // 0 is the shortest falsy value
 
@@ -798,7 +798,7 @@ function loadAudio(url) {
       audioAssets[getName(url)] =
         audioAssets[resolvedUrl] =
         audioAssets[fullUrl] =
-          this;
+        this;
       emit('assetLoaded', this, url);
       resolve(this);
     });
@@ -858,7 +858,7 @@ function loadData(url) {
       dataAssets[getName(url)] =
         dataAssets[resolvedUrl] =
         dataAssets[fullUrl] =
-          response;
+        response;
       emit('assetLoaded', response, url);
       return response;
     });
@@ -897,8 +897,8 @@ function load(...urls) {
       return extension.match(imageRegex)
         ? loadImage(asset)
         : extension.match(audioRegex)
-        ? loadAudio(asset)
-        : loadData(asset);
+          ? loadAudio(asset)
+          : loadData(asset);
     })
   );
 }
@@ -1743,7 +1743,7 @@ class Updatable {
   }
   // @endif
 
-  _pc() {}
+  _pc() { }
 }
 
 /**
@@ -2099,7 +2099,7 @@ class GameObject extends Updatable {
    * @memberof GameObject
    * @function draw
    */
-  draw() {}
+  draw() { }
 
   /**
    * Sync property changes from the parent to the child
@@ -2861,8 +2861,8 @@ class Text extends GameObject {
       textAlign == 'right'
         ? this.width
         : textAlign == 'center'
-        ? (this.width / 2) | 0
-        : 0;
+          ? (this.width / 2) | 0
+          : 0;
     // @endif
 
     this._s.map((str, index) => {
@@ -3112,12 +3112,12 @@ function getCanvasOffset(pointer) {
       rect.left +
       (getPropValue(_s, 'border-left-width') +
         getPropValue(_s, 'padding-left')) *
-        transformScaleX,
+      transformScaleX,
     offsetY:
       rect.top +
       (getPropValue(_s, 'border-top-width') +
         getPropValue(_s, 'padding-top')) *
-        transformScaleY
+      transformScaleY
   };
 }
 
@@ -3757,28 +3757,28 @@ class Button extends Sprite {
    * @memberof Button
    * @function onEnable
    */
-  onEnable() {}
+  onEnable() { }
 
   /**
    * Function called when then button is disabled. Override this function to have the button do something when disabled.
    * @memberof Button
    * @function onDisable
    */
-  onDisable() {}
+  onDisable() { }
 
   /**
    * Function called when then button is focused. Override this function to have the button do something when focused.
    * @memberof Button
    * @function onFocus
    */
-  onFocus() {}
+  onFocus() { }
 
   /**
    * Function called when then button is blurred. Override this function to have the button do something when blurred.
    * @memberof Button
    * @function onBlur
    */
-  onBlur() {}
+  onBlur() { }
 
   onDown() {
     if (!this.disabled) {
@@ -4161,8 +4161,8 @@ function updateGamepad() {
   let pads = navigator.getGamepads
     ? navigator.getGamepads()
     : navigator.webkitGetGamepads
-    ? navigator.webkitGetGamepads
-    : [];
+      ? navigator.webkitGetGamepads
+      : [];
 
   for (let i = 0; i < pads.length; i++) {
     let gamepad = pads[i];
@@ -4509,8 +4509,8 @@ let gestureMap = {
           ? 'left'
           : 'right'
         : y < 0
-        ? 'up'
-        : 'down';
+          ? 'up'
+          : 'down';
     }
   },
   pinch: {
@@ -4845,8 +4845,8 @@ class Grid extends GameObject {
       this.flow == 'column'
         ? 1
         : this.flow == 'row'
-        ? children.length
-        : this.numCols);
+          ? children.length
+          : this.numCols);
 
     let row = 0;
     let col = 0;
@@ -5511,9 +5511,6 @@ function extendObject(kontraObj, properties) {
 
 /**
  * A fast and memory efficient [object pool](https://gameprogrammingpatterns.com/object-pool.html) for sprite reuse. Perfect for particle systems or SHUMPs. The pool starts out with just one object, but will grow in size to accommodate as many objects as are needed.
- *
- * <canvas width="600" height="200" id="pool-example"></canvas>
- * <script src="assets/js/pool.js"></script>
  * @class Pool
  *
  * @param {Object} properties - Properties of the pool.
@@ -5652,7 +5649,7 @@ class Pool {
   update(dt) {
     let obj;
     let doSort = false;
-    for (let i = this.size; i--; ) {
+    for (let i = this.size; i--;) {
       obj = this.objects[i];
 
       obj.update(dt);
@@ -5674,7 +5671,7 @@ class Pool {
    * @function render
    */
   render() {
-    for (let i = this.size; i--; ) {
+    for (let i = this.size; i--;) {
       this.objects[i].render();
     }
   }
@@ -6380,7 +6377,7 @@ class Scene {
    * @memberof Scene
    * @function onShow
    */
-  onShow() {}
+  onShow() { }
 
   /**
    * Function called when the scene is hidden. Override this function to have the scene do something when hidden, such as cleaning up input events.
@@ -6397,7 +6394,7 @@ class Scene {
    * @memberof Scene
    * @function onHide
    */
-  onHide() {}
+  onHide() { }
 }
 
 function factory$2() {
