diff --git a/node_modules/kontra/kontra.mjs b/node_modules/kontra/kontra.mjs
index 356abca..53583cf 100644
--- a/node_modules/kontra/kontra.mjs
+++ b/node_modules/kontra/kontra.mjs
@@ -5374,14 +5374,7 @@ function extendObject(kontraObj, properties) {
 
 /**
  * A fast and memory efficient [object pool](https://gameprogrammingpatterns.com/object-pool.html) for sprite reuse. Perfect for particle systems or SHUMPs. The pool starts out with just one object, but will grow in size to accommodate as many objects as are needed.
- *
- * <canvas width="600" height="200" id="pool-example"></canvas>
- * <script src="assets/js/pool.js"></script>
- * @class Pool
- *
- * @param {Object} properties - Properties of the pool.
- * @param {() => {update: (dt?: Number) => void, render: Function, init: (properties?: Object) => void, isAlive: () => boolean}} properties.create - Function that returns a new object to be added to the pool when there are no more alive objects.
- * @param {Number} [properties.maxSize=1024] - The maximum number of objects allowed in the pool. The pool will never grow beyond this size.
+ 
  */
 class Pool {
   /**
